guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -61,15 +61,19 @@ stage:
 -  **Default**—`true`
 -  **Version**—Magento 2.1.4 and later
 
-Cleans the [generated static view files]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-view.html#config-cli-static-overview) when you perform an action such as enabling or disabling a component. We recommend the default value _true_ in development. The supported values are `true` and `false`.
+Enables or disables cleaning [static content files]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-view.html#config-cli-static-overview) generated during the build or deploy phase. We recommend the default value _true_ in development. If you make modifications to static content through a separate process, set the value to _false_.
+
+-   **`true`**—Removes all existing static content before deploying the updated static content.
+-   **`false`**—The deployment only overwrites existing static content files if the generated content contains a newer version.
 
 ```yaml
 stage:
   deploy:
-    CLEAN_STATIC_FILES: false
+    CLEAN_STATIC_FILES: true
```

Comment:
I believe this is not correct? Billy clarified that default value didn't change, so need to make sure we didn't change it from what is on DevDocs now.

Reply:
Added the sentence about when to set to 'false` before the code sample, and changed value back to `false`.